src/DisplayShortCardData.php

```diff
@@ -1,15 +1,14 @@
 <?php
 
-
 function DisplayShortCardData(array $allCards)
```

Comment:
I missed this, should be return type hinted as string

Reply:
I get red lines under it if I type hin as string?